ui/models/taskcluster.js

```diff
@@ -0,0 +1,142 @@
+import _ from 'lodash';
+import jsone from 'json-e';
+import { Queue, Auth, Hooks } from 'taskcluster-client-web';
+import { satisfiesExpression } from 'taskcluster-lib-scopes';
+
+import taskcluster from '../helpers/taskcluster';
+
+export default class TaskClusterModel {
```

Comment:
We generally lowercase the 'c' in Taskcluster but it is not a big deal either way.

Reply:
Ahh, ok.  I'll try to fix that up as I go from now on.  :)